authenticate/views.py

```diff
@@ -47,12 +48,21 @@ def get(self, request):
         if request.user.is_authenticated:
             return redirect(request.GET["next"])
 
-        # TODO: Only redirect browserish user agents.
+        user_agent_string = request.META.get("HTTP_USER_AGENT")
+        user_agent = parse(user_agent_string)
 
-        return self._redirect(request)
+        if user_agent.browser.family == "Other":
```

Comment:
This is basically there I think, but I'm not sure that this is the best check. It seems like using the derived attributes might be better? E.g. `is_mobile`, `is_tablet`, `is_pc`, `is_touch_capable`, `is_bot`.

Happy to be convinced otherwise though.

Reply:
Seems like the browser family it the only useful bit of information to distinguish, e.g. Chrome from wget, since both can add OS information.